plugins/source/aws/client/spec/spec.go

```diff
@@ -21,7 +21,7 @@ type Spec struct {
 	// In AWS organization mode, CloudQuery will source all accounts underneath automatically.
 	Organization *Organization `json:"org"`
 
-	// If `true`, will log AWS debug logs, including retries and other request/response metadata.
+	// If `true`, will log AWS debug logs, including retries and other request/response metadata. Requires passing `--log-level debug` to the CloudQuery CLI.
```

Comment:
```suggestion
	// If `true`, will log AWS debug logs, including retries and other request/response metadata. Requires debug-level logs to be enabled.
```

This comment might show in managed syncs, where passing parameters to the CLI isn't possible at the moment 🤔 Can we make it a bit more general maybe?

Reply:
Not sure we would expose this in managed syncs (otherwise we'd need to enable CLI debugging for this to work anyways)

Reply:
IMO we need to enable debugging as an extra flag (either debug level or `debug_log: bool`)